test/conftest.py

```diff
@@ -1,15 +1,3 @@
-#   ---------------------------------------------------------------------------------
-#   Copyright (c) Microsoft Corporation. All rights reserved.
-#   Licensed under the MIT License. See LICENSE in project root for information.
-#   ---------------------------------------------------------------------------------
-"""
-This is a configuration file for pytest containing customizations and fixtures.
-
-In VSCode, Code Coverage is recorded in config.xml. Delete this file to reset reporting.
-"""
-
-# https://docs.pytest.org/en/7.1.x/example/markers.html
-# This page outlines the marks available to us
```

Comment:
this is removed since it doesn't apply. This text was left over from a template that was considered for use early on. 99% of the lines of code from said template have been deleted, leaving the dependa-bot git action, a few empty or nearly empty files and the basic directory structure used